recipe/meta.yaml

```diff
@@ -1,43 +1,45 @@
 {% set name = "wandb" %}
-{% set version = "0.13.10" %}
-
+{% set version = "0.13.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wandb-{{ version }}.tar.gz
-  sha256: a2cde45dd6390ed1923ef3e1eca26487cb1e02e4732515856933c4494a975ad3
-
+  sha256: f2b065d1c732e8e1828fabe720dc82f12e6109d0d2bbfbcccc17eaefae0fd7c9
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<38]
+  # skipping s390x as we do not have setproctitle package for this arch. Also, the current package 
+  # will go to Snowflake-specific channel only, so there is no urgency to build it on s390x.
+  skip: true  # [s390x]
```

Comment:
please write here ... [linux and s390x] ... using s390x (or other arch names without OS) might be problematic

Reply:
Done and noted